tox.ini

```diff
@@ -1,27 +1,12 @@
 [tox]
-envlist = py27,py34,py35,py36,py37,py38,flake8,docs
-
-[testenv:py27]
-passenv = PWD TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
-
-deps =
-    {[testenv]deps}
-    nose-cov
-    coverage
-    gevent
-    circus-web
```

Comment:
This means that tests using gevent or circus-web are now skipped. I think we should add a tox env with them. Also, papa tests seems to be skipped, I'm not sure why papa is not installed in tox.

Reply:
After giving it some more thoughts and looking back at the tox config I decided to merge `extra` and main configs, because they are practically the same. The only difference is 1 gevent-related test (`test_daemonize`). Do you think we should have an extra env just for that?

I also realized that with py27 env removed the coverage/coveralls setup broke, so I brought it back in 48dadba.

Reply:
We might have had different envs to make sure that everything was working fine without gevent. But I'm ok with merging this for now, we'll see if it causes issues down the line.